packages/rate-limit/rate-limit.js

```diff
@@ -133,7 +143,10 @@ RateLimiter.prototype.check = function (input) {
   var reply = {
     allowed: true,
     timeToReset: 0,
-    numInvocationsLeft: Infinity
+    numInvocationsLeft: Infinity,
+    userId: input.userId,
+    clientAddress: input.clientAddress,
+    connectionId: input.connectionId,
```

Comment:
@abernix I already made the modification like you suggested in your earlier comment. The callback now has two arguments: the first is the result of the rule (`reply`) and other is the input (`ruleInput`). However I have not pushed the latest code because I am having difficulty with adding test cases for the callback in [this file](https://github.com/meteor/meteor/blob/devel/packages/ddp-rate-limiter/ddp-rate-limiter-tests.js). Callbacks are registered in server so I am uncertain about how to get it result. Could you give me some guidance?